lib/car.rb

```diff
@@ -0,0 +1,33 @@
+class Car
+  attr_reader :make,
+              :model,
+              :monthly_payment,
+              :loan_length
+              :color
+
+  def initialize(make_model, monthly_payment, loan_length)
+    @make = "Ford"
+    @model = "Mustang"
+    @make_model = make_model
+    @monthly_payment = 1500
+    @loan_length = 36
+  end
+
+  def make_model
+   @make << model
+  end
+
+  def total_cost
+    loan_length * monthly_payment
+  end
+
+  def
+    color
+    @color = []
+  end
```

Comment:
Rathering than creating a method for color, we want to create a color attribute that we can have access to:

```
attr_reader :color

def initialize(make_model, monthly_payment, loan_length)
  ....
  @color = nil
end

```

then in our paint method. We can take in a color as a parameter and set our @color 

```
def paint!(color)
  @color = color
end
```